modules/general/acs-email/main.bicep

```diff
@@ -64,6 +70,29 @@ resource email_service 'Microsoft.Communication/emailServices@2023-04-01-preview
   }
 }
 
+resource acs_diagnostic_settings 'microsoft.insights/diagnosticSettings@2016-09-01' = if (enableDiagnostics) {
+  name: 'service'
+  scope: communication_service
+  location: dataLocation
+  properties: {
+    workspaceId: logAnalyticsWorkspaceId
```

Comment:
Can we add an issue to capture that this should probably support blob storage too?  TBH it likely needs a wider review across all the modules to see how consistent our approach is.

Reply:
https://github.com/endjin/Endjin.RecommendedPractices.Bicep/issues/117